docs/src/main/sphinx/sql/show-functions.md

```diff
@@ -18,11 +18,19 @@ For each function returned, the following information is displayed:
 - Deterministic
 - Description
 
+Use the `FROM`  keyword to list user-defined functions in in a specific catalog.
+These functions can be created as [SQL routines](/routines/introduction) or
+[custom function plugins](/develop/functions):
+
+```sql
+SHOW FUNCTIONS FROM example;
```

Comment:
this seems to indicate that functions are in a catalog .. not a schema.. I am not sure what is correct